### ISSUE TITLE
Add CAUTIOUS flag for monsters

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#41]
It could be fun to add very simple bahaivour for friendly townies (eg, animals in started location). Right now they just come closer (after you wake them up) and stick to player. But some of this animals are wild, eg

```
name:deer
base:townsfolk
color:B
speed:110
hit-points:14
hearing:10
armor-class:10
sleepiness:180
depth:0
rarity:3
weight:365
experience:0
flags:MALE
flags:SMART | REGENERATE
flags:NO_FEAR | NO_SLEEP
flags:ANIMAL | FRIENDLY
desc:It doesn't like strangers.
force-location:Hunter's Valley
```

I thought that having smart would make this deer more careful, but she still come close to strangers :)

There could be such solution: to have a new flag like 'FEAR' which would make monster to take distance. Maybe it could be even used for real monsters - so there would be monster type which would have their priority to run from you, no matter did it hurt or not (it should be something flagile, so it won't be annoying).

There is a fruit bat btw:

```
name:fruit bat
base:bat
color:o
speed:120
hit-points:4
hearing:20
armor-class:3
sleepiness:10
depth:1
rarity:1
weight:2
experience:1
blow:BITE:HURT:1d1
flags:DROP_CORPSE | WILD_WOOD | WILD_SWAMP
desc:A fast-moving pest.
```

Not sure why bat likes to fly away so much (or I mixed up it with TomeNet bat?), maybe it's hardcoded?

**Comment by draconisPW** [comment]
Current code makes monsters flee from player when player level is much greater than monster level. I'll add a flag to make them always afraid.

**Comment by draconisPW** [comment]
Implemented in bc7c166.
